src/utils/mod.rs

```diff
@@ -1,3 +1,3 @@
+pub mod data_inserter;
```

Comment:
Why does this have to be a pub mod now?

Reply:
havde gjort den pub fordi jeg også var ved at bøvle med tests samme tid